src/api/app/views/webui/configuration/index.html.haml

```diff
@@ -5,6 +5,10 @@
   = render partial: 'tabs'
   .card-body
     %h3 Configure your Open Build Service instance
+    %p
+      Settings that aren't available here are only available by modifying the
```

Comment:
I guess that after the last changes, all settings are available here, some enabled, some disabled.

I would rewrite this whole sentence to something like:

"Settings that cannot be modified here (they appear as disabled), can be changed by modifying the config/options.yml file of the BsConfig.pm file. Those settings could be modified here if they remove their value in their original configuration file."

Something more in the direction of the error message returned by the API: 

https://github.com/openSUSE/open-build-service/blob/827272c9222aaf702704518b02b714d497e1d177/src/api/app/controllers/configurations_controller.rb#L57

What do you think?

Reply:
I see. I guess that with "that would be incorrect" you refer to my first sentence: "I guess that after the last changes, all settings are available here, some enabled, some disabled.".

The sentence I proposed focuses more on the disabled appearance of the form element. I think it still applies.

Basically, what are you referring to when you say "that should be incorrect"?

Reply:
I mean that none of the settings here are affected by `BSConfig.pm`, and if you look at the screenshot, that kind of notice is already there for fields that are set in `options.yml` in red text

Reply:
Well, that is the point. That notice doesn't inform the administrator, that in order to modify the value from this same form, the value from the `options.yml` file should be removed. Just like the error message of the API.

Reply:
Sure, I improved that notice now